src/middleware/middleware.ts

```diff
@@ -6,6 +6,8 @@ import { debug } from "debug";
 import { IncomingMessage, ServerResponse } from "http";
 import { State, WebhookError, WebhookEventHandlerError } from "../types";
 import { EventNames } from "../generated/event-names";
+
+// @ts-ignore to address #245
 import AggregateError from "aggregate-error";
```

Comment:
Is this import actually used in this file?

Reply:
huh, good catch :)

Reply:
resolved via 034ae6e